performance/BenchmarkRunner.ts

```diff
@@ -0,0 +1,51 @@
+import Benchmark from 'benchmark';
+
+class BenchmarkRunner {
+	private readonly _benchmarks: {
+		benchmark: Benchmark;
+		setup?: () => void;
+		teardown?: () => void;
+	}[] = [];
+
+	public addBenchmark(
+		name: string,
+		test: () => void,
+		setup?: () => void,
+		teardown?: () => void
+	): void {
+		this._benchmarks.push({
+			benchmark: new Benchmark(name, test),
+			// We do not use the setup and teardown which is offered withing the API of benchmarkjs
+			// as several attempts to get this working did not yield any successful results.
```

Comment:
Why not? let's try to make it work

Reply:
I personally don't find this worth the effort. I've done the same solution for fontoxpath and fontoxpath-cache as my attempt to get it working cost me a lot of time and it's very easy to do ourselves.

Reply:
From having a quick look at the Benchmark.js sources, it seems they combine the setup, teardown and test functions using .toString and string concatenation. That would break many things such as closure bindings, so I'm not surprised that can make it hard to get things to work. Let's keep this as is for now (including the comment).